ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/services/admin/PollingUtils.scala

```diff
@@ -64,4 +64,34 @@ object PollingUtils {
     go(1, minWait.min(maxWait).max(50.milliseconds))
   }
 
+  def pollSingleUntilPersisted[T](poll: () => Future[Option[T]])(
```

Comment:
This isn't used anymore so we can drop it for now.

Reply:
Addressed in 9ac958eb4.